github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -5,8 +5,8 @@ presubmits:
     optional: true
     decorate: true
     decoration_config:
-      timeout: 14400000000000 # 4h
-      grace_period: 500000000000 # 5m
+      timeout: 4h # 4h
```

Comment:
nitpicking: we don't need the comments anymore...

Reply:
Done. :+1: